modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -202,7 +206,8 @@ export default class AdvancedOpts {
         innerLabel.merge(innerLabelEnter)
             .attr( 'id', d => `${ d.name }_label` )
             .classed( 'adv-opt-title-disabled', false )
-            .text( d => `${d.label} Options` );
+            .classed( 'adv-opts-group-title', true)
+            .text( d => d.members.length ? `${d.label} Options` : d.label);
```

Comment:
if accordion does not have any advanced options we just don't add the options label